.github/workflows/copilot-build-images.yml

```diff
@@ -0,0 +1,82 @@
+name: copilot-deploy-pipeline
+
+on:
+  push:
+    branches:
+      - "main"
+    tags:
+      - "v*"
+    paths:
+      - "webapi/**"
+      - "webapp/**"
+      - ".github/workflows/copilot-build-images.yml"
+  pull_request:
+    branches:
+      - "main"
+    paths:
+      - "webapi/**"
+      - "webapp/**"
+      - ".github/workflows/copilot-build-images.yml"
```

Comment:
This workflow is great, thank you for contributing! 

For our purposes we don't want to deploy images on every pull request or even on every push to main, but in a more controlled release workflow. If you can change the trigger to `on: workflow_call` that allow it to be called by other workflows and allow anyone to build a pipeline around it.